Requirements.md

```diff
@@ -176,22 +176,15 @@ Cacti requires that the following software is installed on your system.
     - Some of these recommendations may not be applicable depending on the
       version of MySQL/MariaDB you are running.
     - Some of these recommendations should be scaled where appropriate
+    - Newer MySQL/MariaDB databases are using [strict modes](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html)
+      and it can causes unexpected problems with import older Cacti mysqldumps, 
+      like **Can't create table `cacti`.`poller_output_boost` (errno: 140 "Wrong create options")**. 
+      
+      You have more posibilities:
+      - disable appropriate strict mode - not recomended
+      - change mysqldump file - remove **ROW_FORMAT=FIXED** from table definition
+      - before mysqldump run query **ALTER TABLE `poller_output_boost` ROW_FORMAT=DYNAMIC;**   
 
-To implement the above mysql recomendations you can use the below entries and paste them into my.cnf
```

Comment:
You have removed the recommendations, was this for a reason?

Reply:
no, it was mistake

Reply:
It is back